package.json

```diff
@@ -1970,12 +1970,9 @@
                 },
                 "jupyter.pylanceHandlesNotebooks": {
                     "type": "boolean",
-                    "default": false,
-                    "description": "Determines if pylance's experimental notebook support is used or not.",
-                    "scope": "machine",
-                    "tags": [
-                        "experimental"
-                    ]
+                    "default": true,
```

Comment:
Just to check (I've not followed this work as much) is the pylanceHandlesNotebooks = false case still valid going forward? I.E. there is still a reason that a user might opt out of this?

Reply:
I don't believe so.

Is your question, "Should we remove the pylanceHandlesNotebooks = false code path, rather than changing the default"? That's a question for @heejaechang.

FYI, Pylance has three approaches to notebooks at the moment:
1. Defaults -- Jupyter is in charge of concat doc
2. pylanceHandlesNotebooks = true -- Pylance is in charge of concat doc
3. pylanceLspNotebooksEnabled = true -- No concat doc. Separate document for each cell.

Option 3 is an active experiment. I was imagining that we'd wait until that experiment finished to make a decision and come up with a deprecation/removal plan for the other options.

Reply:
Thanks for the explanation. Seems like this is fine for now as long as we handle the overall depreciation plan at some point.